src/contract.rs

```diff
@@ -0,0 +1,84 @@
+use colored::Colorize;
+use ethers::prelude::*;
```

Comment:
nit: avoid "*" includes -- as it makes it harder to see where the given object comes from